### DISCUSSION TITLE
Please correct a word in wiki/Configuration/animations

### DISCUSSION THREAD

**Discussion by moilong** [YaLTeR/niri#349] [General]
https://github.com/YaLTeR/niri/wiki/Configuration:-Animations#window-close
In this section
`animations {
    window-open {
        duration-ms 150
        curve "ease-out-quad"
    }
}`
It should be `animations{
                          window-close{`

**Comment by YaLTeR** [comment]
Good catch, fixed.

By the way, you can just open a pull request against the files in https://github.com/YaLTeR/niri/tree/main/wiki.
